mbuild/utils/io.py

```diff
@@ -196,6 +208,20 @@ def import_(module):
 except ImportError:
     has_hoomd  = False
 
+try:
+    import nglview
+    has_nglview = True
+    del nglview
+except ImportError:
+    has_nglview  = False
```

Comment:
Several of these have two spaces, it seems.
```suggestion
    has_nglview = False
```